### DISCUSSION TITLE
How to build the project with vue.js?

### DISCUSSION THREAD

**Discussion by huangminsky** [gildas-lormeau/zip.js#290] [Q&A] [answered]
my problem:    importing zip.js, an error occurred while compiling Vue project

一、 import * as zip from "@zip.js/zip.js";

<template>
  <div class="hello">
   hello
  </div>
</template>

<script>
import * as zip from "@zip.js/zip.js";
export default {
  name: 'HelloWorld',
  props: {
    msg: String
  },
  mounted() {
    zip.configure({
      useWebWorkers: false
    });
    console.log(zip)
  }
}
</script>

二、 vue.config.js
 module.exports = {
    configureWebpack: {
        module: {
            rules: [
                {
                    test: /\.js$/,
                    use: {
                        loader: '@open-wc/webpack-import-meta-loader',
                    },
                },
            ],
        },
    },
}

三、compile project
  npm run serve
> vue-cli-service build

 WARN  A new version of sass-loader is available. Please upgrade for best experience.
⠼  Building for production...

 ERROR  Failed to compile with 1 error                                                                                              11:01:56 AM

Syntax Error: Thread Loader (Worker 0)
Cannot read property 'length' of undefined

You may use special comments to disable some warnings.
Use // eslint-disable-next-line to ignore the next line.
Use /* eslint-disable */ to ignore all warnings in a file.
 ERROR  Build failed with errors.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! uuutemp@0.1.0 build: `vue-cli-service build`

anyone knows .....???

**Comment by gildas-lormeau** [comment, marked as answer]
I guess that even though you don't use web workers, you have to include the plugin to build the project anyway. This article explains how to install and configure the plugin `worker-plugin`: https://braincoke.fr/blog/2020/03/use-web-workers-with-vue/#worker-plugin.

**Reply by huangminsky** [reply]
I solved this problem by modifying the configuration of webpack！！！Thank you for your guidance
